Makefile

```diff
@@ -72,6 +72,9 @@ golangci-lint:
 	golangci-lint run ./...
 
 test:
+	@if ! [ -a config/simplecontroller.json ]; then \
```

Comment:
According to https://www.tldp.org/LDP/abs/html/fto.html#FTN.AEN3289, `-a` is deprecated for `-e`. Let's use that instead.